packages/playwright-core/src/utils/isomorphic/protocolMetainfo.ts

```diff
@@ -17,7 +17,7 @@
 // This file is generated by generate_channels.js, do not edit manually.
 
 export const methodMetainfo = new Map<string, { internal?: boolean, title?: string, slowMo?: boolean, snapshot?: boolean, pausesBeforeInput?: boolean }>([
-  ['APIRequestContext.fetch', { title: 'Fetch "{url}"', }],
+  ['APIRequestContext.fetch', { title: '{method} "{url}"', }],
```

Comment:
Personal preference:
```suggestion
  ['APIRequestContext.fetch', { title: 'Fetch {method} "{url}"', }],
```

So its clear its Browser vs. Fetch but I also understanding that Fetch is often used alone - so then its probably noise. Fine for me.